docs/source/quickstart.md

```diff
@@ -1,21 +1,70 @@
 # Quickstart
 
+## Connecting to an Ethereum node
+
+An ETH RPC provider needs to be specified via the `-ethUrl` flag when starting `livepeer` in order to connect to mainnet or Rinkeby.
+
+If you do not want to run your own ETH node, you can use services such as [Infura](https://infura.io/) or [Alchemy](https://alchemyapi.io/) that host ETH nodes on behalf of users.
+
+For example, you could use the following command to connect an orchestrator to Infura.
+
+```
+$ livepeer -network rinkeby -orchestrator -pricePerUnit 1 -ethUrl https://rinkeby.infura.io/v3/<PROJECT_ID>
+```
+
+See the [Infura docs](https://infura.io/docs/gettingStarted/makeRequests.md) for more details on how to obtain a `<PROJECT_ID>`.
+
+If you would like to run your own ETH node you can use `geth` (the Rinkeby testnet is only supported by `geth`) which can be installed using the instructions on the [installing geth page](https://geth.ethereum.org/docs/install-and-build/installing-geth). 
```

Comment:
> the Rinkeby testnet is only supported by `geth`

Does this mean "geth is the only Ethereum implementation that works on Rinkeby", or "geth only works on Rinkeby" ?

Reply:
Supposed to say that geth is the only Ethereum implementation that works on Rinkeby.

Clarified in 95e52c5

Reply:
As far as I'm aware most clients support the rinkeby test net. 

The parity client has a different clique implementation and can thus not connect to the rinkeby test net, however the parity client is no longer being actively developed by Parity the company so I'm hesitant to still list it as an option for our users. 
Legacy users will  upgrade as they see fit but we shouldn't be pointing new users towards it. 

I propose replacing this paragraph with a list of clients one can use, eg:

If you would like to run your own Ethereum node you can use one of these clients:
- [Geth](https://geth.ethereum.org/downloads/)
- [Hyperledger Besu](https://besu.hyperledger.org/en/stable/)
- [Trinity](https://trinity.ethereum.org/)
- [Nethermind](http://nethermind.io/client)
- [Aleth](https://github.com/ethereum/aleth) 

We can then put the geth example underneath as a subsection for this.

Reply:
Ah cool didn't realize that other clients support Rinkeby now - nice!

Added a sentence about other clients in c884aad that links out to this [EthHub page](https://docs.ethhub.io/using-ethereum/running-an-ethereum-node/).